docs/install_guides/installing-openvino-linux.md

```diff
@@ -240,11 +240,6 @@ cd /opt/intel/openvino_2022/tools/model_optimizer/install_prerequisites
    sudo ./install_prerequisites_caffe.sh
    ```
 
-   - For **TensorFlow 1.x**:
-   ```sh
-   sudo ./install_prerequisites_tf.sh
-   ```
-
    - For **TensorFlow 2.x**:
    ```sh
    sudo ./install_prerequisites_tf2.sh
```

Comment:
please write it without particular indicated version as: 
```
For **TensorFlow**: 
sudo ./install_prerequisites_tf.sh
```

We will avoid confusion as if TF1 models no longer supported by MO

Reply:
Done.